library/src/main/java/com/datatorrent/lib/io/WebSocketInputOperator.java

```diff
@@ -207,7 +207,6 @@ public Thread newThread(Runnable r)
         @Override
         public void onMessage(String string)
         {
-          LOG.debug("Got: " + string);
```

Comment:
An accidental removal of the debug message or was it intentional?

Reply:
I thought we should remove it but I can put it back if you insist.

Reply:
No strong opinion either way just wanted to check it was not accidental removal. Wouldn't having the ability to see the received message be useful for debugging or are there other ways to get to it.